Runtime configuration not supported anymore

Release 1.6.7 breaks our integration, which had been working fine since 1.1
Root cause is "Resource 'sdk_config.properties' could not be found"

We use merchantsdk which includes the "LATEST" paypal-core. We use runtime configuration: we do not have any sdk_config.properties, because we are a software provider for many merchants. Their credentials are populated at runtime. 


Stack trace:
```
Caused by: java.lang.ExceptionInInitializerError
        at com.paypal.core.DefaultHttpConnection.<init>(DefaultHttpConnection.java:35)
        at com.paypal.core.ConnectionManager.getConnection(ConnectionManager.java:52)
        at com.paypal.core.APIService.makeRequestUsing(APIService.java:152)
        at com.paypal.core.BaseService.call(BaseService.java:265)
        at urn.ebay.api.PayPalAPI.PayPalAPIInterfaceServiceService.doDirectPayment(PayPalAPIInterfaceServiceService.java:2415)
        at urn.ebay.api.PayPalAPI.PayPalAPIInterfaceServiceService.doDirectPayment(PayPalAPIInterfaceServiceService.java:2367)
        ... 105 more
Caused by: java.lang.RuntimeException: java.io.IOException: Resource 'sdk_config.properties' could not be found
        at com.paypal.core.ConfigManager.<init>(ConfigManager.java:91)
        at com.paypal.core.ConfigManager.getInstance(ConfigManager.java:103)
        at com.paypal.core.SSLUtil.<clinit>(SSLUtil.java:43)
        ... 140 more
Caused by: java.io.IOException: Resource 'sdk_config.properties' could not be found
        at com.paypal.sdk.util.ResourceLoader.getInputStream(ResourceLoader.java:50)
        at com.paypal.core.ConfigManager.<init>(ConfigManager.java:86)
        ... 142 more
```

--- a/src/main/java/com/paypal/core/ConfigManager.java
+++ b/src/main/java/com/paypal/core/ConfigManager.java
@@ -24,7 +24,7 @@ import com.paypal.sdk.util.ResourceLoader;
  * 
  */
 public final class ConfigManager {
-
+	
 	/**
 	 * Singleton instance variable
 	 */
@@ -64,6 +64,7 @@ public final class ConfigManager {
 		DEFAULT_PROPERTIES.put(Constants.HTTP_CONNECTION_MAX_CONNECTION, "100");
 		DEFAULT_PROPERTIES.put(Constants.DEVICE_IP_ADDRESS, "127.0.0.1");
 		DEFAULT_PROPERTIES.put(Constants.GOOGLE_APP_ENGINE, "false");
+		DEFAULT_PROPERTIES.put(Constants.SSLUTIL_JRE, "SunJSSE");
 		DEFAULT_PROPERTIES.put(Constants.SSLUTIL_PROTOCOL, "TLS");	
 		defaultMapView = new HashMap<String, String>();
 		for (Object object : DEFAULT_PROPERTIES.keySet()) {
@@ -82,13 +83,15 @@ public final class ConfigManager {
 		 */
 		ResourceLoader resourceLoader = new ResourceLoader(
 				Constants.DEFAULT_CONFIGURATION_FILE);
+		properties = new Properties();
 		try {
 			InputStream inputStream = resourceLoader.getInputStream();
-			properties = new Properties();
 			properties.load(inputStream);
-			setPropertyLoaded(true);
 		} catch (IOException e) {
-			throw new RuntimeException(e);
+			// We tried reading the config, but it seems like you dont have it. Skipping...
+			LoggingManager.info(ConfigManager.class, Constants.DEFAULT_CONFIGURATION_FILE + " not present. Skipping...");
+		} finally {
+			setPropertyLoaded(true);
 		}
 	}
 
@@ -142,7 +145,7 @@ public final class ConfigManager {
 				combinedProperties.load(new ByteArrayInputStream(bos
 						.toByteArray()));
 			} catch (IOException e) {
-				// TODO return defaultProperties
+				// Something failed trying to load the properties. Skipping...
 			}
 		}
 		return combinedProperties;

